src/components/TodoComponents/TodoList.js

```diff
@@ -1,2 +1,14 @@
-// your components will all go in this `component` directory.
-// feel free to change this component.js into TodoList.js
+import React from "react";
+import Todo from "./Todo";
+
+const List = props => {
+  return (
+    <div>
+      {props.todoData.map((item, index) => {
+        return <Todo task={item} key={item.task} taskIndex={index} toggleTodo={props.toggleTodo} />;
```

Comment:
The key attribute must be unique. If I enter the same task twice, the key will not be unique and it will lead to bugs in the UI.

Reply:
That's something I knew, but didn't implement it there.  I have changed it to id which is unique because it uses Date.now() to create a unique ID for every new take added.  Date.now was already implemented on my addNewTodo item in App.js for that reason.

Thanks for the feedback 👍